packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -558,7 +559,7 @@ class PaginatedDataTableState extends State<PaginatedDataTable> {
               ? null
               : _handleLast,
         ),
-      Container(width: 14.0),
```

Comment:
Hm, I wonder why the `sized_box_for_whitespace` lint didn't trigger on these. They seem to fit pretty squarely into the lint's territory: https://dart.dev/tools/linter-rules/sized_box_for_whitespace

Reply:
I believe the lint only triggers when the Container has a `child`, since switching away from the default `LimitedBox` child could potentially result in different behavior.